velox/tool/trace/TraceReplayRunner.cpp

```diff
@@ -346,8 +353,6 @@ void TraceReplayRunner::run() {
   }
 
   VELOX_USER_CHECK(!FLAGS_task_id.empty(), "--task_id must be provided");
```

Comment:
Can we move
```
  VELOX_USER_CHECK(!FLAGS_root_dir.empty(), "--root_dir must be provided");
  VELOX_USER_CHECK(!FLAGS_query_id.empty(), "--query_id must be provided");
  VELOX_USER_CHECK(!FLAGS_node_id.empty(), "--node_id must be provided");
```
Into TraceReplayRunner::init() and set fs_ as a class member there? thanks!